index.js

```diff
@@ -275,6 +278,8 @@ AFRAME.registerComponent('environment', {
         this.hemilight.setAttribute('light', {'intensity': 0.6});
       }
       else {
+        var skycol = new THREE.Color('#CEE4F0');
+        this.hemilight.setAttribute('light', {'color': '#' + skycol.getHexString()});
```

Comment:
Are those lines necessary? When I remove them I don't see any change.
The color is already set in
https://github.com/supermedium/aframe-environment-component/blob/858bc1084ddb18a84593a0d5c68b216e123ee42f/index.js#L189-L193

Reply:
Ah the previous code is in init, your added code is in update, so is was using maybe a previous environment color when you changed the environment settings, is that it?
The change here has nothing to do with color conversion right?
`skycol.getHexString()` is giving me back `#CEE4F0` so we can simplify here by
`this.hemilight.setAttribute('light', {'color': '#CEE4F0'});` right?

Reply:
Indeed, there was a subtle bug where `atmosphere` sky types didn't explicitly set the `hemilight` colour, meaning it would be the last set value, which could be the default value or, in case of switching between presets, any other colour from any of the presets.

I seem to remember that there was some reason to not use a literal constant, but with all the changes to colour management in Three and A-Frame since, I don't remember. In any case, the colour value _is_ sRGB and the light component does expect sRGB values, so it's totally fine. Will change it